aws/resource_aws_iam_access_key.go

```diff
@@ -28,6 +29,7 @@ func resourceAwsIamAccessKey() *schema.Resource {
 			},
 			"status": {
```

Comment:
Can you please add a validation function to this attribute? 👍 

```go
"status": {
  Type:     schema.TypeString,
  Optional: true,
  Computed: true,
  ValidateFunc: validation.StringInSlice([]string{
    iam.StatusTypeActive,
    iam.StatusTypeInactive,
  }, false),
},
```

Reply:
Done - thanks for the pointer!